src/clojure_lsp/source_paths.clj

```diff
@@ -49,6 +49,13 @@
          set
          (set/union root-source-paths))))
 
+(defn relative-to-deps-file [deps-file]
+  (let [p (.getParentFile (io/file deps-file))]
+    (fn [f]
+      (if (.isAbsolute (io/file f))
+        f
+        (shared/normalize-file (io/file p f))))))
+
```

Comment:
We can't have reflections because of graalvm, also would you mind changing to:
```clojure
(defn ^:private relative-to-deps-file [file ^java.io.File deps-file]
  (if (.isAbsolute (io/file file))
    file
    (shared/normalize-file (io/file p f))))
```
and use the function on a anonymous function on the caller?